prime-router/metadata/fhir_mapping/hl7/segments/ORC/DiagnosticReport.yml

```diff
@@ -238,14 +238,22 @@ basedOn:
   valueOf: datatype/Reference
   expressionType: resource
   specs: $ServiceRequest
+  generateList: true
   useGroup: true
 
 specimen:
-  valueOf: datatype/Reference
+  expressionType: nested
   generateList: true
-  expressionType: resource
-  specs: $Specimen
-  useGroup: true
+  expressions:
+    - valueOf: datatype/Reference
+      generateList: true
+      expressionType: resource
+      specs: $Specimen
+      useGroup: true
+    - valueOf: datatype/Reference
+      expressionType: resource
+      specs: $SpecimenSource
```

Comment:
Ok, I see why the name change from above; makes sense to me, but maybe include this in the ticket for fixing up the library because ideally we wouldn't need to do this.